java/examples/src/main/java/glide/examples/ExamplesApp.java

```diff
@@ -2,6 +2,7 @@
 package glide.examples;
 
 import glide.api.RedisClient;
+import glide.api.models.commands.LPosOptions;
```

Comment:
running `./gradlew spotlessApply` should get rid of unused imports and will fix formatting issues